shared/modules/selectors.ts

```diff
@@ -0,0 +1,137 @@
+import { ALLOWED_SMART_TRANSACTIONS_CHAIN_IDS } from '../constants/smartTransactions';
```

Comment:
The idea of this shared file is to not duplicate code between UI and background, especially these selectors:
getSmartTransactionsEnabled
getIsSmartTransaction